DTS-STN/Service-Canada-Design-System
# Update footer.js auth state to match new design

# DS-##Footer.js

Main Reviewer: @zeddotes 

#### Description & Background Context:

A new design was submitted for the auth state of the footer. You can view it here:
https://www.figma.com/file/YFz6bzcX1DSAQfu4BSfm6G/Extended-GC-Design-Library---DECD?node-id=6594%3A22869&t=tG852iUEt2iyTXyL-0

The component has been modified to reflect this new design. 

#### Where should the reviewer start?

File name: Footer.js

#### How should this be tested?

Run the storybook and view the component, see that it matches the specifications for font, spacing, sizing and responsiveness in the figma file.

#### Are there any breaking changes?

The Contact Us link and link text has been moved out of the array from which the link list is generated for the auth state to it's own object.
